frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -29,17 +30,25 @@ class Circuit:
     configuration: Configuration
 
     graph: Graph
-    mlir: str
```

Comment:
I think we should keep this as str, it's a breaking change to change it and in memory module is not anyway.